jekyll/_ccie/overview.md

```diff
@@ -1,33 +1,52 @@
 ---
 layout: enterprise
 section: enterprise
-title: "Clustered Installation Overview"
-category: [installation]
+title: "Overview"
+category: [documentation]
 order: 1
-description: "High-level overview of the CircleCI Enterprise Installation process."
+description: "High-level overview of the CircleCI Enterprise."
 ---
 
-This article provides a platform-independent overview of CircleCI Enterprise. CircleCI Enterprise matches the experience of <https://circleci.com> but runs behind your organization's firewall, allowing your code, builds, and production hosts to be inaccessible outside of your network.
+This article provides an overview of CircleCI Enterprise architecture in the following sections:
 
-We are constantly working on making the installation process as smooth as possible and expanding the administrative tooling.  We appreciate your feedback on ways to make CircleCI Enterprise easier and more valuable for you and your team, so please contact us at <enterprise-support@circleci.com> with any suggestions.
+* TOC
+{:toc}
 
+CircleCI is a modern continuous integration and continuous delivery (CI/CD) platform. The CircleCI Enterprise solution is installable inside your private cloud or data center and is free to try for two weeks. CircleCI automates build, test, and deployment of software.
 
-### Installation Steps
+## Basic Features
 
-Once you have all of the prerequisites in place, you can either [follow the detailed
-installation steps for AWS]({{site.baseurl}}/enterprise/aws/), or [for environments not on AWS]({{site.baseurl}}/enterprise/docker-install/).
+After a software repository in GitHub Enterprise is added as a project to the CircleCI Enterprise application, every new commit triggers a build and notification of success or failure through webhooks with integrations for Slack, HipChat, Campfire, Flowdock, or IRC notifications. Code coverage results are available from the details page for any project for which a reporting library is added.
 
+CircleCI may also be configured to deploy code to various environments, including the following:
+
+- AWS CodeDeploy
+- AWS EC2 Container Service (ECS)
+- AWS S3
+- Google Container Engine (GKE)
+- Heroku
+
+Other cloud service deployments can be easily scripted using SSH or by installing the API client of the service with your job configuration.
+
+Contact us at <enterprise-support@circleci.com> to request a trial license.
+
+## Build Environments
+
+CircleCI Enterprise installations may run a single Linux or MacOS
+image for builds. Users of your CircleCI Enterprise installation must use this single contianer image for all of their builds and may **not** choose between Precise and Trusty build container images, or custom built
+images, for example. 
+
+**Note:** If your CircleCI Enterprise installation is not open to the internet, you may
+need to connect to a VPN or take other special steps to access the CircleCI application or SSH builds.
```

Comment:
I'm not sure I follow here -- most CCIE customers will likely presume they're setting it up in their own VPC or other private cloud that isn't necessarily "open to the Internet" but is open to their personnel -- this paragraph might make it sound like we are introducing some kind of need to have a VPN setup.